docs/source/snippets/config/data/image/folder/classification/cli/default.yaml

```diff
@@ -0,0 +1,22 @@
+class_path: anomalib.data.Folder
```

Comment:
since we introduced folder.yaml in configs, can we use that instead of this? Might reduce the number of places we will have to update in case of any changes.

Reply:
`symlink` might be a good idea. Would it also work on Windows? Does sphinx support linking to a different directory? Maybe we can provide the path to the configs folder where this file is referred. That is if its not too complex. Otherwise we can keep this file as is.

Reply:
not sure, here is how it would look like if we refer to the original config file 
`../../../../../../src/configs/data/folder.yaml`

Reply:
Hmm alright. Let's keep this as is